doc/advanced-configuration/Listener-modules.md

```diff
@@ -197,3 +198,18 @@ An example would be [`mod_disco`](../modules/mod_disco.md), which may be configu
 A reason to do so could be reduced traffic - systems with many components could return very long disco responses.
 Also, some deployments would like to avoid revealing some services; not because it is a security threat (this method does not prevent clients from communicating with hidden components), but rather because they are not meant to interact with clients directly (e.g. helper components for other components).
 
+## Conflict behaviour
+
+By default, when a component tries to connect and a registration conflict occures, we drop such connection by sending:
+
+```xml
+<stream:error>
+  <conflict xmlns='urn:ietf:params:xml:ns:xmpp-streams'/>
+</stream:error>
+</stream:stream>
+```
+
+It makes implementing reconnection logic difficult, because an old connection would not allow any other connections.
```

Comment:
It makes implementing the reconnection logic difficult, because the old connection would not allow any other connections.